codebase/patches/simplesaml_config.patch

```diff
@@ -731,8 +731,8 @@ index 00000000..399dde26
 +     * Options: [syslog,file,errorlog,stderr]
 +     *
 +     */
-+    'logging.level' => SimpleSAML\Logger::DEBUG,
-+    'logging.handler' => 'syslog',
++    'logging.level' => SimpleSAML\Logger::INFO,
```

Comment:
Can/should this and the one below it be setup as env vars?  In a kubernetes environment, we would want the handler below set to stderr and if we need to debug something it would be nice to be able to just change the env var to debug on the fly.

Reply:
Good catch, that can be changed!

Reply:
@jhujasonw this has been parameterized; if there's nothing further you can approve the PR (in order to get the green checkmark) and I'll merge.